### ISSUE TITLE
[BUG] Cannot connect to 192.168.0.1

### ISSUE THREAD

**Issue by teod0r** [anthonycr/Lightning-Browser#605]
net::ERR_CONNECTION_REFUSED

**Comment by anthonycr** [comment]
Are you trying to use a proxy like I2P or Orbot? Or are you just trying to connect directly to an IP address?

**Comment by teod0r** [comment]
I don't use any proxy, trying connect directly, typed 192.168.0.1 in address bar.
I have I2P and orbot installed, but no proxy selected in Lightning settings.

**Comment by anthonycr** [comment]
I've been testing this and it works for me as expected. Since `192.168.0.1` will point to the default IP address of the router, if that isn't your router's IP address then you will get the connection refused error. I tested on a router for which this was the default IP and Lightning successfully navigated to the page. I tested on a router that was not using default settings and I got the connection refused message as expected.

If something is wrong in Lightning and it isn't simply that there's nothing at the IP address, then another browser should be able to navigate to the IP address. Try it out in another browser and let me know if it works.
